pkg/serverless/daemon/utils_test.go

```diff
@@ -0,0 +1,29 @@
+// +build !race
```

Comment:
why do we need this tag?

Reply:
I don't think we actually do -- I left this in accidentally from doing some testing. Removing.